docs/decisions/0002-extracts-plugins-infrastructure-from-edx-platform.rst

```diff
@@ -0,0 +1,24 @@
+1. Extract plugins Infrastructure from edx-platform
+===================================================
+
+Status
+------
+
+Draft
+
+Context
+-------
+
+We have experimented with adding plugins to edx-platform and are now ready to move this functionality to other IDAs. However, since the plugins infrastructure is currently located in edx-platform and thus not easily usable by IDAs.
```

Comment:
1. The sentence starting `However...` needs to either drop the word `since`, or adjust the words `and thus`.
2. Call it `Django plugins`.
3. Add another sentence that states what Django plugins are. This probably exists somewhere, and you could copy and paste.